_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -0,0 +1,192 @@
+
+There are a lot of places for good, solid advice on the document you are using to persuade someone to interview you. So I'll tell you what has worked for me. I think it should be a one page document, that captures a snapshot of you. Things they must have: your name, a way to contact you and a list of things you think is relevant. Ideally it's going to be one page. 
+
+You should consider what you think are qualities that you bring to the table, for some job applications I have submitted both a resume and a design portfolio. If you are particularly proud of your design work, perhaps find a way to include your app store screenshots and branding?
+
+
+##### On Representing Myself
+
+I dug through my archives and found every resume I have ever created; ranging from my first as a student, through to last year.
+
+- [2007](/images/2016-01-30-iOS-Junior-Interviews/2007-144.jpg) as a part of the WWDC Student Scholarship. It was significantly less competitive then.
+- [2008](/images/2016-01-30-iOS-Junior-Interviews/2008-general-144.jpg) as a graduate applying for jobs that were not programming focused.
+- [2008](/images/2016-01-30-iOS-Junior-Interviews/2008-programming-144.jpg) as a graduate applying for programming jobs.
+- [2010](/images/2016-01-30-iOS-Junior-Interviews/2010-144.jpg) when applying for jobs once I had stopped being a junior.
+- [2015](/images/2016-01-30-iOS-Junior-Interviews/2015-144.jpg) my CV as a part of my U.S. VISA process.
+
+You can definitely get a sense of my skill specialisations happening over the last 8 years, but the tone of how I present myself hasn't really changed.
+
+I also consider my personal web-page [orta.io](http://orta.io) to act as a portfolio, Ash Furrow's portfolio page has a [similar vibe](https://ashfurrow.com/portfolio/).
+
+##### On the team 
+
+Want to know what the resumes looked like for people applying to Artsy? I asked some of our team to send me their last resumes
+
+<table>
+  <thead>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+			<a href="/images/2016-01-30-iOS-Junior-Interviews/ash-furrow-resume.pdf">
+				<img height="200" src="/images/2016-01-30-iOS-Junior-Interviews/ash-thumbnail.jpg"></br>
+				Ash Furrow (2013)
+			</a>
+		</td>
+      <td>
+			<a href="/images/2016-01-30-iOS-Junior-Interviews/sarah-scott-resume.pdf">
+			<img height="200" src="/images/2016-01-30-iOS-Junior-Interviews/sarah-scott-resume.jpg"><br/>
+			Sarah Scott (2014)
+			</a>
+		</td>
+      <td>
+			<a href="https://ortastuff.s3.amazonaws.com/artsy/maximcramer_artsy_portfolio.pdf">
+				<img height="200" src="/images/2016-01-30-iOS-Junior-Interviews/maximcramer_artsy_portfolio_thumb.jpg"></br>
+				Maxim Cramer (2015)
+			</a>
+		</td>
+    </tr>
+  </tbody>
+</table>
+
+&nbsp;
+
+A portfolio can be whatever you want it to be. You could spend forever on a portfolio, but it's really just an exercise in restraint and prioritisation. Eventually it's been shaved to a point where you can remove nothing more.
```

Comment:
> just an exercise in restraint and prioritisation

I love this phrase a lot, and upon reflection, it's very germane. Programmers need to have both skills.